tests/benchmarks/igraph_set.c

```diff
@@ -0,0 +1,64 @@
+#include <igraph.h>
+
+#include "bench.h"
+#define TOSTR1(x) #x
+#define TOSTR(x) TOSTR1(x)
+#define BARBASI_GRAPH_VERTICE_COUNT 100000
+
+void unit_test_subgraph_edges(
+    const igraph_t *graph, igraph_vector_int_t *vertices,
+    igraph_integer_t induced_subgraph_vertice_count, igraph_integer_t benchmark_count,
+    const char* bench_message
+) {
+    igraph_vector_int_t edges;
+    printf("bench called\n");
+
+    ;
+    BENCH(bench_message,
+          REPEAT(
+                igraph_induced_subgraph_edges(graph, igraph_vss_range(0, induced_subgraph_vertice_count),&edges), 
```

Comment:
You are using `edges` without initializing it first.

Reply:
Can't believe I still made that mistake. Thank you